src/sage/categories/additive_magmas.py

```diff
@@ -911,14 +917,14 @@ def _neg_(self):
 
                         EXAMPLES::
 
-                           sage: x = cartesian_product((GF(7)(2),17)) ; x
+                           sage: x = cartesian_product((GF(7)(2), 17)); x                                               # optional - sage.rings.finite_rings
                            (2, 17)
-                           sage: -x
+                           sage: -x                                                                                     # optional - sage.rings.finite_rings
                            (5, -17)
 
                         TESTS::
 
-                           sage: x.parent() in AdditiveMagmas().AdditiveUnital().AdditiveInverse().CartesianProducts()
+                           sage: x.parent() in AdditiveMagmas().AdditiveUnital().AdditiveInverse().CartesianProducts()  # optional - sage.rings.finite_rings
```

Comment:
```suggestion
                           sage: C = AdditiveMagmas().AdditiveUnital().AdditiveInverse().CartesianProducts() 
                           sage: x.parent() in C                                       # optional - sage.rings.finite_rings
```
This allows you to place the # at column 88, and you can align the two previous labels.